utils/wasm-builder/src/cargo_command.rs

```diff
@@ -36,7 +37,8 @@ impl CargoCommand {
         CargoCommand {
             path: "cargo".to_string(),
             manifest_path: "Cargo.toml".into(),
-            args: vec!["+nightly", "rustc", "--target=wasm32-unknown-unknown"],
+            toolchain: "nightly",
```

Comment:
Looks like this could be done in separate PR

Reply:
Agree, will do it later.